bicepconfig.json

```diff
@@ -5,7 +5,7 @@
 		"dynamicTypeLoading": true
 	},
 	"extensions": {
-		"radius": "br:biceptypes.azurecr.io/radius:latest",
-		"aws": "br:biceptypes.azurecr.io/aws:latest"
+		"radius": "br:biceptypes.azurecr.io/radius:v0.38.0-rc1",
+		"aws": "br:biceptypes.azurecr.io/aws:v0.38.0-rc1"
```

Comment:
@sk593 is this the right PR to do this in? do you forsee any issues?

Reply:
no, this doesnt get updated. the version that lives in the repo will always be latest to correspond with the latest version of radius

Reply:
we auto-generate the bicepconfig during app setup, so that's when this would get populated with a specific release version

Reply:
kk, removing then